features/lint-code.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Lint code using Tuist (SwiftLint)
+
+    Scenario: The project is an iOS application where all targets pass linting (app_with_framework_and_tests_passing_swiftlint)
```

Comment:
Rather than having 2 projects, one where Swiftlint passes, and another one where Swiftlint doesn't passes, I'd have one project where one target passes and the other one doesn't. The reason being is that fixtures have a maintenance cost that we should avoid as much as possible. One project is more maintainable than 2, and considering these scenarios can be reduced into a single project, I'd rather go for that.